### DISCUSSION TITLE
Homebridge Podman no space left on device

### DISCUSSION THREAD

**Discussion by bozza-man** [unifi-utilities/unifi-common#471] [General]
Hey Guys,

Am I missing something here. Do I need to find a new mounting point / expand?

`Resolving "oznu/homebridge" using unqualified-search registries (/etc/containers/registries.conf)
Trying to pull docker.io/oznu/homebridge:latest...
Getting image source signatures
Copying blob f9c8c4881a5e skipped: already exists
Copying blob f04b4bbe1580 skipped: already exists
Copying blob 77301028049d skipped: already exists
Copying blob c57e54085274 skipped: already exists
Copying blob 29e216a2e539 skipped: already exists
Copying blob e70cc601de9a skipped: already exists
Copying blob f1f94e96dfd3 done
Error: writing blob: adding layer with blob "sha256:f1f94e96dfd37bcb3bc4c2528743cfcf2698c0007310de0d74e45246ca323e84": error creating layer with ID "51a51d8224bcff0c839c12d151ab8d7ee2e69880f608b486cea85bab5cc4de9f": no space left on device`

df -h
Filesystem                         Size  Used Avail Use% Mounted on
udev                               2.0G     0  2.0G   0% /dev
tmpfs                              395M   31M  365M   8% /run
/dev/disk/by-partlabel/root        2.0G  1.7G  163M  92% /boot/firmware
/dev/loop0                         833M  833M     0 100% /mnt/.rofs
/dev/disk/by-partlabel/overlay     9.4G  8.7G  212M  98% /mnt/.rwfs
overlayfs-root                     9.4G  8.7G  212M  98% /
/dev/disk/by-partlabel/log         976M  162M  748M  18% /var/log
/dev/disk/by-partlabel/persistent  2.0G  280M  1.6G  16% /persistent
tmpfs                              2.0G  108K  2.0G   1% /dev/shm
tmpfs                              5.0M     0  5.0M   0% /run/lock
tmpfs                              2.0G     0  2.0G   0% /sys/fs/cgroup
tmpfs                              987M   72K  987M   1% /tmp
/dev/sda1                          1.8T  1.2G  1.8T   1% /volume1
tmpfs                               16M  204K   16M   2% /var/log/ulog
shm                                 63M  4.9M   58M   8% /var/lib/containers/storage/vfs-containers/20bc02a3d1f64bdc3db44b8493b388ec0123ac5fc802588e0316a17324a54431/userdata/shm
tmpfs                              512M     0  512M   0% /var/opt/unifi-protect/tmp
tmpfs                               64M     0   64M   0% /var/opt/unifi/tmp

**Comment by vneno** [comment]
probably problem with vfs driver. 
Since You have external drive (sda1) it should be easy.
I suggest somethink like:
- create dir /volume1/vfs (mkdir /volume1/vfs)
- stop all containers (podman stop XXX)
- mv containers data to /volume1/vfs (mv /var/lib/containers/storage/vfs/* /volume1/vfs)
- create on_boot.d script which should start before any podman container 
- this script shold bind the path  /volume1/vfs to  /var/lib/containers/storage/vfs (mount --bind  /volume1/vfs /var/lib/containers/storage/vfs)
- run/start/pull container/image
